editor/src/clj/editor/properties_view.clj

```diff
@@ -300,9 +300,9 @@
                 cancel-fn (fn [_]
                             (let [property (property-fn)
                                   current-vals (properties/values property)]
-                              (update-ui-fn current-vals (properties/validation-message property)
-                                            (properties/read-only? property))
-                              ))
+                              (update-ui-fn current-vals
+                                            (properties/validation-message property)
+                                            (properties/read-only? property))))
```

Comment:
Fixed formatting.

Reply:
I use 
![CleanShot 2024-06-28 at 22 07 45@2x](https://github.com/defold/defold/assets/2209596/df65bbfd-2806-497f-a20e-4e8c5556b099)
everywhere I touch something. Not sure if it did something wrong or I just missed this part